tests/neurotechdevkit/scenarios/test_results.py

```diff
@@ -489,3 +489,35 @@ def test_creation_of_pulsed_3d_results_sliced(result_args, recorded_slice):
     )
     result = create_pulsed_result(**result_args)
     assert isinstance(result, PulsedResult3D)
+
+
+@pytest.mark.parametrize("slice_args", [(3, 0.0), (None, 0.0), (1, None), (1, 0.5)])
+def test_validate_slice_args(fake_ss_result_3d, a_test_scenario_3d, slice_args):
+    """Validates that raises error if trying to perform invalid slice"""
+    fake_ss_result_3d.scenario = a_test_scenario_3d
+    with pytest.raises(ValueError):
+        fake_ss_result_3d._validate_slice_args(*slice_args)
```

Comment:
nit (i.e. minor suggestion):
```suggestion
@pytest.mark.parametrize("slice_axis,slice_position", [(3, 0.0), (None, 0.0), (1, None), (1, 0.5)])
def test_validate_slice_args(fake_ss_result_3d, a_test_scenario_3d, slice_axis, slice_position):
    """Validates that raises error if trying to perform invalid slice"""
    fake_ss_result_3d.scenario = a_test_scenario_3d
    with pytest.raises(ValueError):
        fake_ss_result_3d._validate_slice_args(slice_axis=slice_axis, slice_position=slice_position)
```